dangerfile.js

```diff
@@ -115,7 +115,17 @@ function setReleaseDates() {
 
 		if ( nextMilestone ) {
 			jetpackReleaseDate = moment( nextMilestone.due_on ).format( 'LL' );
-			codeFreezeDate = moment( nextMilestone.due_on ).subtract( 7, 'd' ).format( 'LL' );
+
+			// Look for a code freeze date in the milestone description.
+			const dateRegex = /\d{4}[/-]?(0?[1-9]|1[012])[/-]?(0?[1-9]|[12][0-9]|3[01])$/;
```

Comment:
I see a few issues here:

* This regex will accept things like "2020/1-2" or "202012"  that `moment()` won't accept.
* Also this fails to find the date if there's an extra newline or whitespace after the date. Or any other content, for that matter.
* I note this will also pick up any date in the description and assume it's a code freeze date, which might not be what we want.

I'd look for `/^Code Freeze: (\d{4}-\d{2}-\d{2})\s*$/m` specifically, to find a line like `Code Freeze: 2020-10-26`. Like a header or footer depending on where you put it in the description.

Or that approach would allow us to safely match `/^Code Freeze: (.*)\s*$/m` and accept anything where `moment( ... ).isValid()` is true.

Reply:
I like that, that's a lot simpler. c945660d55d23340166a6c68bbc366ab4d4a960b